pkgs/development/cuda-modules/lib/default.nix

```diff
@@ -0,0 +1,13 @@
+let
+  lib = import ../../../../lib;
+in
+lib.fixedPoints.makeExtensible (final: {
```

Comment:
Side note: do I understand correctly that with auto-called args the `import ./default.nix { }` would always be a think even though the function object itself is in the import cache?

Reply:
I'm afraid I don't understand with respect to "auto-called args" and the snippet you have with `import` -- are you talking about `callPackage`?

Reply:
I meant the top-level lambda args ("auto-called" as in nix-build/nix-shell applying `{}` to lambdas).

Basically, the question is, do we benefit from import cache in case of `{ foobar ? ... }:`, or only in case of `let foobar = ...; in`?